redis/controller.cc

```diff
@@ -49,7 +49,7 @@ future<> controller::listen(seastar::sharded<auth::service>& auth_service, db::c
     return utils::resolve(cfg.rpc_address, family, preferred).then([this, server, &cfg, keepalive, ceo = std::move(ceo), &auth_service] (seastar::net::inet_address ip) {
         auto get_config = sharded_parameter([&] {
             return redis_transport::redis_server_config {
-                ._timeout_config = make_timeout_config(cfg),
+                ._timeout_config = updated_timeout_config(cfg),
```

Comment:
> Nitpick: Since we already have the name **updateable** value, no **updated** value, I suggest that we should use the prefix updatable_ everywhere, not updated_.

the same suggestion was proposed by @xemul , see https://github.com/scylladb/scylladb/pull/12531#discussion_r1144522389 . and i still think you guys suggestions make sense.

yeah, the explanation helps to convince me that updateable is better. thanks. the naming is difficult.